### ISSUE TITLE
Is Circular Layout supported by Vis.js

### ISSUE THREAD

**Issue by ositowang** [almende/vis#3947]
Hi, All:
  I am using vis.js to visualize a process map these days. I have a question regarding the graph layout.
  Is there any circular layout supported  by vis.js?
  I red some posts on the internet. Some said no, but other says yes(without codes provided)
  I want to know if circular layout is possible with vis.js or not.

**Comment by KenyStev** [comment]
Do you mean circular layout the way you connect nodes of the network?
if it is then yes
just go ahead and check the models documentation for an example like this: http://visjs.org/examples/network/edgeStyles/colors.html

**Comment by ositowang** [comment]
Hi Keny:
  Thank you for replying. What I mean is not nodes style, but network graph layout. Generally, we create a network from vis.js like this:
![image](https://user-images.githubusercontent.com/26860856/39257947-c2ecebb6-4880-11e8-9d4d-2fef44ff091c.png)
However, for a process map, I prefer it in a circular layout., like this one:
![image](https://user-images.githubusercontent.com/26860856/39258049-fc4c6cec-4880-11e8-8f89-24ad2f8dac3a.png)
Is this layout supported by vis.js?

**Comment by emorling** [comment]
I am also curious about this and specifically like this:

[https://4.bp.blogspot.com/-r5msw9K3sUQ/Vt9IjRphm0I/AAAAAAAAADU/manLDdm9yAY/s1600/Screenshot_2016-03-07-21-22-57.png](url)

**Comment by mahit777** [comment]
I don't think so that these layouts are supported yet in Network Module. Only Layout option is Hierarchial and the default is radial
